ynh-dev

```diff
@@ -99,7 +102,7 @@ elif [ "$1" = "create-env" ]; then
 elif [ "$1" = "run" ]; then
 
     DOMAIN=$2
-    VERSION='stable'
+    VERSION='stable64'
```

Comment:
I can't agree more, as 32bits will be slowly discontinued, and 64bits is de facto standard,